run.sh

```diff
@@ -59,6 +59,7 @@ case "$deployment_action" in
         build_images
         ;;
     "deploy_osh")
+        patch_upstream
```

Comment:
this doesn't need to happen here. We have enough interfaces to cover that -- Run patch_upstream, build_images, deploy_osh if you want to do it manually, or run setup_everything if you want to deploy everything.

We could add an interface setup_openstack, which would be the things happening after "setup_hosts".

Reply:
I understand why this was done, and therefore I implemented a new interface in https://github.com/SUSE-Cloud/socok8s/pull/93 .